### ISSUE TITLE
[Packetbeat] - Change field names to follow ECS

### ISSUE THREAD

**Issue by andrewkroh** [elastic/beats#7968]
For the next major release we want to change some of the common field names in Packetbeat to follow the [Elastic Common Schema](https://github.com/elastic/ecs#fields) (ECS). This will make it much easier to query data related data from different sources.

We should break this task down into smaller pieces that are easily reviewable. Perhaps something like this.

- Map the fields to ECS in a spreadsheet and review.
  - Map the common fields that shared by all protocols (e.g. `ip`, `client_ip`, etc.)
  - Map Packetbeat flow data
  - Map the protocols that are commonly used in security analytics (e.g. dns, tls, http, icmp).
  - Check the application protocols to see if there are any fields that can be mapped to ECS.
- Convert one protocol at a time including updating dashboards.

**Comment by andrewkroh** [comment]
@MikePaquette @webmat I have not added `event.kind`, `event.category`, or `event.action` fields to anything in Packetbeat.

I'm assuming we will want:

- `event.kind` = event 
- `event.category` = network_traffic

For `event.action` I'm not sure what to use. Do we want something that's protocol specific like "dns-query", "http-post", "myqsl-select"?

**Comment by webmat** [comment]
I haven't had time to look into event categorization much, yet. What's important for GA is to get all the breaking changes squared away.

Event categorization can be considered gradual additions, during the 7.x line. Conversely, rushing to get an answer for these fields may be painful later.

The exception may be `event.kind`, where Packetbeat should have `event` across the board. But the other two, I'm fine with waiting.

**Comment by MikePaquette** [comment]
@andrewkroh @webmat 

- I am 👍 for `event.kind` = event
- I am also in favor of going ahead and populating `event.category` = network_traffic.  Even though we warn users that we may define values for this field, I know I'l be pushing for "network_traffic" to be one of those values.
- Also in favor of populating `event.action`.  I was thinking something more generic, but I like your idea of the protocol-specific values.  (although the field is named `event.action`, we are typically using it more like as if it were `event.activity`) and those protocol specific values would fit.  Did @adriansr populate these fields in the filebeat netflow input?  We should be consistent.

**Comment by webmat** [comment]
Yeah the consistency is specifically why I say we should wait after 7.0 FF, when we have the time to have a look around, consider all of our existing modules / sources, as well as others we expect to support shortly.

Adding those after 7.0 is perfectly fine. Changing the value after the fact (if we get it wrong in a rush) is a breaking change (although a small one).

**Comment by andrewkroh** [comment]
I opened #10357 to add:

- event.kind = event
- event.category = network_traffic (already used by Filebeat netflow)
- event.action = network_flow - for Packetbeat flow events only (already used by Filebeat netflow)

We can add `event.action` to other protocols later.

**Comment by MikePaquette** [comment]
👍
